src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/service/ReferralServiceIntegrationTest.kt

```diff
@@ -0,0 +1,206 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.integration.service
```

Comment:
ReferralServiceIntegrationTest already exists ```uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.ReferralServiceTest```. Could you please port your test there